src/ServiceControl.UnitTests/API/APIApprovals.cs

```diff
@@ -24,14 +24,20 @@ class APIApprovals
         [Test]
         public void PublicClr()
         {
-            var publicApi = ApiGenerator.GeneratePublicApi(typeof(Bootstrapper).Assembly);
+            var publicApi = typeof(Bootstrapper).Assembly.GeneratePublicApi(new ApiGeneratorOptions
+            {
+                ExcludeAttributes = new[] { "System.Reflection.AssemblyMetadataAttribute" }
```

Comment:
I'm aware that we previously did not remove the target framework either but I'm wondering why that would need to be in the approval file

Reply:
This TF is not modifying _what_ is verified. We strictly stick to fixing the issue of OublicApiApprovals.